packages/nimble-components/src/table/tests/table-keyboard-navigation.spec.ts

```diff
@@ -0,0 +1,1059 @@
+            await waitForUpdatesAsync();
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('with the first row focused, clicking a link in another row will focus that link, and an Esc key press will refocus the cell', async () => {
+            await sendKeyPressesToTable(keyArrowDown, keyArrowLeft); // focus row
+
+            const row2Anchor = pageObject.getRenderedCellAnchor(1, 0);
+            await clickAndSendFocusEvents(row2Anchor);
+
+            expect(currentFocusedElement()).toBe(row2Anchor);
+
+            await sendKeyPressToTable(keyEscape);
+
+            expect(currentFocusedElement()).toBe(pageObject.getCell(1, 0));
+        });
+
+        describe('and cell[0, 1] focused,', () => {
+            beforeEach(async () => {
+                await sendKeyPressesToTable(keyArrowDown, keyArrowRight);
+            });
+
+            it('each Tab press will focus each of the remaining tabbable elements in the row (to the right), then will move focus past the table', async () => {
+                const expectedFocusedElements = [
+                    pageObject.getRenderedCellAnchor(0, 1),
+                    pageObject.getCellActionMenu(0, 1)!,
+                    pageObject.getRenderedCellAnchor(0, 2)
+                ];
+                for (const expectedElement of expectedFocusedElements) {
+                    const tabEvent = await sendKeyPressToTable(keyTab);
+                    expect(currentFocusedElement()).toBe(expectedElement);
+                    expect(tabEvent.defaultPrevented).toBe(true);
+                }
+                const lastTabEvent = await sendKeyPressToTable(keyTab);
+                await verifyLastTabKeyEventBehavior(lastTabEvent);
+            });
+
+            it('each Shift-Tab press will focus each of the remaining tabbable elements in the row (to the left), then will move focus past the table', async () => {
+                const expectedFocusedElements = [
+                    pageObject.getCellActionMenu(0, 1)!,
+                    pageObject.getRenderedCellAnchor(0, 1),
+                    pageObject.getRenderedCellAnchor(0, 0)
+                ];
+                for (const expectedElement of expectedFocusedElements) {
+                    const tabEvent = await sendKeyPressToTable(keyTab, {
+                        shiftKey: true
+                    });
+                    expect(currentFocusedElement()).toBe(expectedElement);
+                    expect(tabEvent.defaultPrevented).toBe(true);
+                }
+                const lastTabEvent = await sendKeyPressToTable(keyTab, {
+                    shiftKey: true
+                });
+                await verifyLastTabKeyEventBehavior(lastTabEvent);
+            });
+
+            const focusContentKeys = [keyEnter, keyFunction2];
+            focusContentKeys.forEach(key => {
+                it(`pressing ${key} will focus the anchor in the current cell`, async () => {
```

Comment:
Can we use `parameterize` instead of a `forEach` around an `it` or does something about this pattern make that a challenge?

Same thing a few other places in the file.

Reply:
Using `parameterizeSpec` now.